.circleci/config.yml

```diff
@@ -26,6 +26,7 @@ commands:
       - node/install-packages
       - run: npm run build
       - run: npm run lint
+      - run: npx prettier --check "packages/**/src/**/*.ts"
```

Comment:
I wonder if we should add this to the `lint` npm script?  @ukstv in case he has an opinion here.

Reply:
Yup. I am all for prettier. It should be part of `lint` npm script, so that a developer gets it automatically, without any extra commands.

Reply:
It could be `npx lerna run lint && npx prettier ...` inside root `package.json`.